wiki/Modding/en.md

```diff
@@ -1,98 +1,237 @@
 
-### [Song Setup](Song_Setup "wikilink")
+If you spot these, it is a noteworthy thing to comment about in a mod post.
 
-The actual Song Setup options do not impact game play, but it is important to give correct names. The artist name, the title of the song, the source (if applicable), and the tags should all be the same in each difficulty. Audio Lead-in times are usually the same, but can differ depending on when the first note is played. The "Enable Countdown" button should be either activated or deactivated in all difficulties.
+### Awkward Slider Velocity Changes
 
-### Background, Skin, and Combo Colors
+_See also: [Slider](/wiki/Slider)._
 
-Most importantly, all beatmaps must have a background picture. The skin should not blend in with the background or greatly hinder game play (because of complex [hitcircle](hitcircle "wikilink") design, huge hit bursts, etc.). There are also many [Skinning](Skinning "wikilink") rules which are needed to be followed. Similar to Skinning, combo colors should not blend in with the background either. These combo colors should also be very distinct and clear. The mapper should not have two almost identical combo colors (unless they are separated by a very contrasting color in between).
+Sometimes maps contain slider velocity changes that may not make much sense or are very hard to follow.
+If a slider velocity change requires more than just intuition to understand, then it is usually not a fitting slider velocity change.
+A common example of when to use a slider velocity change would be in a section of a song that is slowed down or more calming than the rest of the song.
+Here, many mappers would likely slow down the slider velocity to match the calmed music.
 
-### [Storyboarding](Storyboarding "wikilink")
+### Hit Sounds
 
-If anything weird in the storyboard happens (bad rendering of pictures, emptiness, etc.) it is a good idea to tell the storyboarder (who may or may not be the mapper) so that he or she can fix it. The Storyboard generally should not exceed an S.B. Load of 5.0x. You can check out the Storyboard by pressing F2 or clicking the "Design" tab at the top of the screen in [editor](editor "wikilink"). The dimensions of the elements should not exceed the default size.
+_See also: [Hit Sounds](/wiki/Hit_Sounds)._
 
-### Preview Point
+Hit sounds are the noise the game makes when the player successfully taps a [hit object](/wiki/hit_object).
+All maps should have a sufficient amount of hit sounds whether using custom or default hit sounds; otherwise, they cannot be ranked.
+
+Sometimes, hit sounds can sound awkward or weird or they are too quiet or too loud.
+Cases like these may confuse the player.
+
+Using a volume that cannot be heard, in most cases, is unrankable.
+Something like this is noteworthy to pay attention and point out.
+
+### Kiai Time
+
+_See also: [Kiai Time](/wiki/Kiai_Time)._
+
+Kiai time should be used when make sense and should not be abused.
```

Comment:
this doesnt explain anything + "Kiai time should be used when **it makes** sense"
here's some actual reasons as to why kiai time shouldn't be abused:
- as kiai time triggers a fountain of stars on its start and makes all objects flash repetitively overusing it can actually strain the eyes of a player and make for a generally unpleasant experience
   - kiai toggling frequently should be avoided for this reason, it's kind-of-bearable in osu mode but in taiko the entire interface will flash up and thus toggling kiai often should be avoided there
- Kiai time, by definition, signifies that one part of the song is more important so using it everywhere loses its meaning
- if nothing in the section can justify kiai time being used on it, then kiai time should not be used (i.e. randomly spamming the toggle)